js/arbeidsrettet-oppfolging/Mann.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
```

Comment:
Det kan være lurt å kjøre `svgo` eller noe annet for å komprimere svgen før du legger den ved her, da tar den litt mindre plass

Reply:
fikser!